app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
+  end
+
+  def import(dry_run = false)
```

Comment:
thoughts on naming this `run` ? we are doing that (somewhat inconsistently) in other service classes